BaseDockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian
```

Comment:
2 things here
* you are using the wrong image, below you're installing a lot of python packages, rather use something like the tensorflow image

* The dockerfile probably isn't nessary to be honest. Whilst using multiple images is fairly common this is generally because you have the same settings for a number of images. In this case best practice is to have a single base image that is publically available on dockerhub, that image is then pulled and written to. The reason why you will generally have a publicly available image is because that base if that base image is compromised so is everything else so if you're not comfortable having it public then it probably shouldn't be there.

The other problem with having a base image is that if you update it then you need to rebuild everything so where possible try to avoid having one